### PR TITLE
 Improve documentation (code style, release)

### PR DESCRIPTION
**Pull request by tgonzalezorlandoarm**
- Add code style information
- Define the release expectations

Signed-off-by: Tomás González tomasagustin.gonzalezorlando@arm.com

### CODE REVIEW

File: src/parsec_service/tests/README.md [hunk 1]
```diff
@@ -16,9 +16,10 @@ multithreaded client. Valid requests are sent intermittently so as to check that
 still up and working correctly.
 
 The `ci.sh` script executes all tests and is used on the CI.
-[`rustfmt`](https://github.com/rust-lang/rustfmt) and
-[`clippy`](https://github.com/rust-lang/rust-clippy) are needed for code formatting and static
-checks.
+
+Parsec's code style is enforced by [`rustfmt`](https://github.com/rust-lang/rustfmt) and
+[`clippy`](https://github.com/rust-lang/rust-clippy), which are needed for code formatting and
```
Comment (gowthamsk-arm) [inline src/parsec_service/tests/README.md:21]: I think this needs to be changed, clippy is only for static lint checks.

Comment (tgonzalezorlandoarm) [reply src/parsec_service/tests/README.md:21]: Done. I think that is clearer to what I meant :)

**Review by ionut-arm — APPROVED** [review]
💯

**Review by gowthamsk-arm — APPROVED** [review]
LGTM. Thanks

**Pull request merged**